readthedocs/invitations/templates/invitations/email/invitation.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'core/email/common.html' %}
+{% block content %}
+<p>
+  {{ from_name }} has invited you to join the {{ object_description }}.
```

Comment:
```suggestion
  {{ from_name }} has invited you to join {{ object_description }}.
```

Reply:
GitHub does include the `the` in their messages

![](https://user-images.githubusercontent.com/1140183/181655043-478e77f9-a99b-4f55-b795-6131f5fa860b.png)

/cc @agjohnson

Reply:
It's mostly fine either way, but I think with `the` sounds the best. Without `the`, this will read:

```Santos has invited you to join Read the Docs organization```